lib/dotenv.ex

```diff
@@ -43,6 +43,21 @@ defmodule Dotenv do
     \z
     /x
 
+  # https://regex101.com/r/XrvCwE/1
+  @env_expand_pattern ~r/
+    (?:^|[^\\])                           # prevent to expand \\$
+    (                                     # get variable key parttern
```

Comment:
This looks reasonable. Do you mind fixing this typo, then I'll merge?

Reply:
sorry for the delay. if you are not to make direct to the master, I can make another PR with the fix

Reply:
I'll merge the fix if you PR it, but it's no big deal if you don't have time.

Reply:
done https://github.com/avdi/dotenv_elixir/pull/33